From 17c881710fe2ef374b4da4c3eb60b6eedd5600d0 Mon Sep 17 00:00:00 2001
From: Rj <rj.constantino@easyshop.ph>
Date: Fri, 28 Aug 2015 13:38:20 +0800
Subject: [PATCH 07/27] Enhanced codes.

---
 .../resources/ViewController/ProductViewController.swift         | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/YiLinkerOnlineBuyer/resources/ViewController/ProductViewController.swift b/YiLinkerOnlineBuyer/resources/ViewController/ProductViewController.swift
index 5858ae1..1a0846a 100644
--- a/YiLinkerOnlineBuyer/resources/ViewController/ProductViewController.swift
+++ b/YiLinkerOnlineBuyer/resources/ViewController/ProductViewController.swift
@@ -394,13 +394,8 @@ class ProductViewController: UIViewController, ProductImagesViewDelegate, Produc
             (task: NSURLSessionDataTask!, responseObject: AnyObject!) in
             
             SVProgressHUD.dismiss()
-            
-            if (responseObject["isSuccessful"] as! Bool) {
-                SessionManager.parseTokensFromResponseObject(responseObject as! NSDictionary)
-                self.requestUpdateWishlistItem()
-            } else {
-                self.showAlert(title: "Error", message: responseObject["message"] as! String)
-            }
+            SessionManager.parseTokensFromResponseObject(responseObject as! NSDictionary)
+            self.requestUpdateWishlistItem()
             
             }, failure: {
                 (task: NSURLSessionDataTask!, error: NSError!) in
-- 
2.3.2 (Apple Git-55)

