From ac5fc0793727009d71d7eb7c2aaa0e41cf3e4d90 Mon Sep 17 00:00:00 2001
From: Rj <rj.constantino@easyshop.ph>
Date: Fri, 28 Aug 2015 14:12:55 +0800
Subject: [PATCH 10/27] Optimized followed seller ang category page.

---
 .../FollowedSellerViewController.swift             |  8 ------
 .../ViewController/HiddenViewController.swift      | 32 ++++++++++++++++------
 2 files changed, 24 insertions(+), 16 deletions(-)

diff --git a/YiLinkerOnlineBuyer/resources/ViewController/FollowedSellerViewController.swift b/YiLinkerOnlineBuyer/resources/ViewController/FollowedSellerViewController.swift
index 7cef33b..d779440 100644
--- a/YiLinkerOnlineBuyer/resources/ViewController/FollowedSellerViewController.swift
+++ b/YiLinkerOnlineBuyer/resources/ViewController/FollowedSellerViewController.swift
@@ -113,16 +113,8 @@ class FollowedSellerViewController: UIViewController, EmptyViewDelegate {
         manager.POST(url, parameters: params, success: {
             (task: NSURLSessionDataTask!, responseObject: AnyObject!) in
             
-            if (responseObject["isSuccessful"] as! Bool) {
                 SessionManager.parseTokensFromResponseObject(responseObject as! NSDictionary)
                 self.requestFollowedSelers()
-            } else {
-                SVProgressHUD.dismiss()
-                let alertController = UIAlertController(title: "Error", message: "message", preferredStyle: .Alert)
-                let defaultAction = UIAlertAction(title: "OK", style: .Default, handler: nil)
-                alertController.addAction(defaultAction)
-                self.presentViewController(alertController, animated: true, completion: nil)
-            }
             
             }, failure: {
                 (task: NSURLSessionDataTask!, error: NSError!) in
diff --git a/YiLinkerOnlineBuyer/resources/ViewController/HiddenViewController.swift b/YiLinkerOnlineBuyer/resources/ViewController/HiddenViewController.swift
index c39c447..4e2c018 100644
--- a/YiLinkerOnlineBuyer/resources/ViewController/HiddenViewController.swift
+++ b/YiLinkerOnlineBuyer/resources/ViewController/HiddenViewController.swift
@@ -47,7 +47,14 @@ class HiddenViewController: UIViewController {
         if index == 0 {
             
         } else if index == 1 {
-            
+            if SessionManager.isLoggedIn() {
+                var titleLabel = UILabel(frame: CGRectZero)
+                titleLabel.font = UIFont(name: "Panton-Regular", size: 20)
+                titleLabel.text = "Followed Sellers"
+                titleLabel.textColor = .whiteColor()
+                titleLabel.sizeToFit()
+                self.navigationItem.titleView = titleLabel
+            }
         } else if index == 2 {
             
         } else if index == 3 {
@@ -62,21 +69,30 @@ class HiddenViewController: UIViewController {
                 self.navigationItem.titleView = titleLabel
             }
         } else if index == 5 {
-            if SessionManager.isLoggedIn() {
+            if !SessionManager.isLoggedIn() {
                 var titleLabel = UILabel(frame: CGRectZero)
                 titleLabel.text = "Customize Shopping"
                 titleLabel.font = UIFont(name: "Panton-Regular", size: 20)
                 titleLabel.textColor = .whiteColor()
                 titleLabel.sizeToFit()
                 self.navigationItem.titleView = titleLabel
+            } else if SessionManager.isLoggedIn() {
+                var titleLabel = UILabel(frame: CGRectZero)
+                titleLabel.font = UIFont(name: "Panton-Regular", size: 20)
+                titleLabel.text = "Category Page"
+                titleLabel.textColor = .whiteColor()
+                titleLabel.sizeToFit()
+                self.navigationItem.titleView = titleLabel
             }
         } else if index == 6 {
-            var titleLabel = UILabel(frame: CGRectZero)
-            titleLabel.font = UIFont(name: "Panton-Regular", size: 20)
-            titleLabel.text = "Category Page"
-            titleLabel.textColor = .whiteColor()
-            titleLabel.sizeToFit()
-            self.navigationItem.titleView = titleLabel
+            if !SessionManager.isLoggedIn() {
+                var titleLabel = UILabel(frame: CGRectZero)
+                titleLabel.font = UIFont(name: "Panton-Regular", size: 20)
+                titleLabel.text = "Category Page"
+                titleLabel.textColor = .whiteColor()
+                titleLabel.sizeToFit()
+                self.navigationItem.titleView = titleLabel
+            }
         }
         
     }
-- 
2.3.2 (Apple Git-55)

