From 1197f1de82f2e13a1e1643e15862b994a6a1b9d3 Mon Sep 17 00:00:00 2001
From: jpChan15 <john.chan@easyshop.ph>
Date: Sat, 29 Aug 2015 14:03:03 +0800
Subject: [PATCH 17/27] Added badge to cart.

---
 .../resources/ViewController/CartViewController.swift               | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/YiLinkerOnlineBuyer/resources/ViewController/CartViewController.swift b/YiLinkerOnlineBuyer/resources/ViewController/CartViewController.swift
index 20a27bd..5a25c0e 100644
--- a/YiLinkerOnlineBuyer/resources/ViewController/CartViewController.swift
+++ b/YiLinkerOnlineBuyer/resources/ViewController/CartViewController.swift
@@ -142,7 +142,11 @@ class CartViewController: UIViewController, UITableViewDelegate, UITableViewData
         self.updateCounterLabel()
         self.calculateTotalPrice()
         self.dismissLoader()
-       
+        
+        if tableData.count != 0 {
+            let badgeValue = (self.tabBarController!.tabBar.items![4] as! UITabBarItem).badgeValue?.toInt()
+            (self.tabBarController!.tabBar.items![4] as! UITabBarItem).badgeValue = String(tableData.count)
+        }
     }
     
     func updateCounterLabel() {
-- 
2.3.2 (Apple Git-55)

