From 4fd8fd6bc2d5c63bb72142b9557482a177cb6851 Mon Sep 17 00:00:00 2001
From: jpChan15 <john.chan@easyshop.ph>
Date: Sat, 29 Aug 2015 20:49:37 +0800
Subject: [PATCH 19/27] Resolved app crashes when retrieving wihlist data from
 web.

---
 .../resources/Model/WishlistProductDetailsModel.swift               | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/YiLinkerOnlineBuyer/resources/Model/WishlistProductDetailsModel.swift b/YiLinkerOnlineBuyer/resources/Model/WishlistProductDetailsModel.swift
index 45aeeff..0ab7475 100644
--- a/YiLinkerOnlineBuyer/resources/Model/WishlistProductDetailsModel.swift
+++ b/YiLinkerOnlineBuyer/resources/Model/WishlistProductDetailsModel.swift
@@ -91,8 +91,10 @@ class WishlistProductDetailsModel {
                 image = tempVar
             }
             
-            if let tempVar = dictionary["images"] as? NSArray {
-                images = tempVar as! [String]
+            for tempVar in dictionary["images"] as! NSArray {
+                if let temp = tempVar["imageLocation"] as? String {
+                    images.append(temp)
+                }
             }
             
             if let tempVar = dictionary["shortDescription"] as? String {
-- 
2.3.2 (Apple Git-55)

