From e6f2cf20ac3183dcd5f1ac3d04cee646e9a6757b Mon Sep 17 00:00:00 2001
From: jpChan15 <john.chan@easyshop.ph>
Date: Sat, 29 Aug 2015 20:52:03 +0800
Subject: [PATCH 20/27] Added badge to wishlist.

---
 .../resources/ViewController/WishlistViewController.swift          | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/YiLinkerOnlineBuyer/resources/ViewController/WishlistViewController.swift b/YiLinkerOnlineBuyer/resources/ViewController/WishlistViewController.swift
index 3e116f1..e481462 100644
--- a/YiLinkerOnlineBuyer/resources/ViewController/WishlistViewController.swift
+++ b/YiLinkerOnlineBuyer/resources/ViewController/WishlistViewController.swift
@@ -149,6 +149,13 @@ class WishlistViewController: UIViewController, UITableViewDelegate, UITableView
         }
         self.updateCounterLabel()
         self.dismissLoader()
+        
+        if tableData.count != 0 {
+            let badgeValue = (self.tabBarController!.tabBar.items![3] as! UITabBarItem).badgeValue?.toInt()
+            (self.tabBarController!.tabBar.items![3] as! UITabBarItem).badgeValue = String(tableData.count)
+        } else {
+            (self.tabBarController!.tabBar.items![3] as! UITabBarItem).badgeValue = nil
+        }
     }
     
     func updateCounterLabel() {
-- 
2.3.2 (Apple Git-55)

