From 379adf93cf43d57e147165edf10b2b44150af1a9 Mon Sep 17 00:00:00 2001
From: jpChan15 <john.chan@easyshop.ph>
Date: Mon, 31 Aug 2015 12:38:10 +0800
Subject: [PATCH 25/27] Fixed app crashes upon accessing Profile.

---
 YiLinkerOnlineBuyer/resources/Model/AddressModelV2.swift | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/YiLinkerOnlineBuyer/resources/Model/AddressModelV2.swift b/YiLinkerOnlineBuyer/resources/Model/AddressModelV2.swift
index 0d5be65..0cb95e5 100644
--- a/YiLinkerOnlineBuyer/resources/Model/AddressModelV2.swift
+++ b/YiLinkerOnlineBuyer/resources/Model/AddressModelV2.swift
@@ -15,7 +15,7 @@ class AddressModelV2 {
     var title: String = ""
     var unitNumber: String = ""
     var buildingName: String = ""
-    var streetNumber: Int = 0
+    var streetNumber: String = ""
     var streetName: String = ""
     var subdivision: String = ""
     var zipCode: String = ""
@@ -70,7 +70,7 @@ class AddressModelV2 {
             
             if let value: AnyObject = reviews["streetNumber"] {
                 if value as! NSObject != NSNull() {
-                    model.streetNumber = value as! Int
+                    model.streetNumber = value as! String
                 }
             }
             
-- 
2.3.2 (Apple Git-55)

