From a8230804fe2b2afe56acfcd39b8cd11b8ee3a7d7 Mon Sep 17 00:00:00 2001
From: Yhel <joriel.fronda@easyshop.ph>
Date: Mon, 31 Aug 2015 13:47:25 +0800
Subject: [PATCH 27/27] Resolved conflict.

---
 YiLinkerOnlineBuyer/resources/View/ProductReviewFooterView.swift | 1 -
 1 file changed, 1 deletion(-)

diff --git a/YiLinkerOnlineBuyer/resources/View/ProductReviewFooterView.swift b/YiLinkerOnlineBuyer/resources/View/ProductReviewFooterView.swift
index 06e98c9..f54b422 100644
--- a/YiLinkerOnlineBuyer/resources/View/ProductReviewFooterView.swift
+++ b/YiLinkerOnlineBuyer/resources/View/ProductReviewFooterView.swift
@@ -10,7 +10,6 @@ import UIKit
 
 protocol ProductReviewFooterViewDelegate {
     func seeMoreReview(controller: ProductReviewFooterView)
-    func getProductReviewFooterView()
 }
 
 class ProductReviewFooterView: UIView {
-- 
2.3.2 (Apple Git-55)

